src/main/java/controller/GetUser.java

```diff
@@ -47,7 +47,7 @@ public void getUserWithoutUserIdPathVariable(HttpServletRequest request) {
      * Returns details of the requested User.
      */
     @GetMapping("/users/{userId}")
-    public ImmutableMap<String, Object> getUser(@PathVariable("userId") String userIdString, HttpServletRequest request) {
+    public ImmutableMap<String, Map<String, Object>> getUser(@PathVariable("userId") String userIdString, HttpServletRequest request) {
 
         String path = request.getRequestURI();
         Map<String, Object> responseBody;
```

Comment:
Cant we make this immutable?

Reply:
Oops, we aren't using this variable anywhere. This got left here by mistake. Will remove this.

Reply:
Removed